src/RF24LogLevel.h

```diff
@@ -0,0 +1,42 @@
+/**
+ * @file RF24LogLevel.h
+ *
+ * Created on: 2 Oct 2020
+ *     Author: Witold Markowski (wmarkow)
+ *
+ * Copyright (C)
+ *    2020        Witold Markowski (wmarkow)
+ *    2021        Brendan Doherty (2bndy5)
+ *
+ * This General Public License does not permit incorporating your program into
+ * proprietary programs.  If your program is a subroutine library, you may
+ * consider it more useful to permit linking proprietary applications with the
+ * library.  If this is what you want to do, use the GNU Lesser General
+ * Public License instead of this License.
+ */
+
+#ifndef SRC_RF24LOGLEVEL_H_
+#define SRC_RF24LOGLEVEL_H_
+
+#include <avr/pgmspace.h>

```

Comment:
could probably replace this with `#include "stdint.h"` if all you need here is the `uint8_t`

Reply:
Yes,you are right. The class will be decoupled from AVR.

Reply:
Done.